controllers/virtualmachine/volumebatch/volumebatch_controller.go

```diff
@@ -264,7 +264,7 @@ func (r *Reconciler) ReconcileNormal(ctx *pkgctx.VolumeContext) error {
 		}
 	}
 
-	legacyAttachments, err := r.getAttachmentsForVM(ctx)
+	legacyAttachments, err := pkgutil.GetAttachmentsForVM(ctx, r.Client, ctx.VM)
```

Comment:
Nit: this is kind of a generic name in a generic package - can we use the full type name?